packages/components/src/components/switch/switch.stories.ts

```diff
@@ -21,151 +25,44 @@ export default {
+ * */
 
-export const Checked = {
-  parameters: { controls: { exclude: ['checked'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        }
-      },
-      constants: { type: 'attribute', name: 'checked', value: true },
-      args
-    });
-  }
+export const Required = {
+  render: () => html`<sd-switch required>Required</sd-switch>`
 };
 
 /**
- * Test invalid state inside a form.
+ * The component gets `invalid` state when the form is not valid.
  */
 
 export const Invalid = {
-  parameters: { controls: { exclude: ['required'] } },
-  render: (args: any) => {
-    return html`
-      <form>
-        ${generateTemplate({
-          args,
-          constants: [{ type: 'attribute', name: 'required', value: true }]
-        })}
-        <sd-button style="margin-top: 16px" type="submit">Submit</sd-button>
-      </form>
-    `;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('sd-button');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
-};
-
-/**
- * Use the `base`, `control--unchecked`, `control--checked`, `checked` and `label` part selectors to customize the switch.
- */
-
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: [
-        'base',
-        'control',
-        'control--unchecked',
-        'control--checked',
-        'label',
-        'title',
-        'name',
-        'value',
-        'size',
-        'disabled',
-        'checked',
-        'form',
-        'thumb',
-        'required',
-        'default'
-      ]
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-switch::part(...){outline: solid 2px red}',
-          values: ['base', 'control', 'control--unchecked', 'control--checked', 'thumb', 'label'].map(part => {
-            return {
-              title: part,
-              value: `
-                <style>
-                    #part-${part} sd-switch::part(${part}){outline: solid 2px red};
-                    .hidden {display: none}
-                </style>
-                
-                <div id="part-${part}">${
-                  part.includes('control--checked')
-                    ? '<sd-switch checked>Default Slot</sd-switch>'
-                    : '<sd-switch>Default Slot</sd-switch>'
-                }</div>
-                <div class="hidden">%TEMPLATE%</div>
-              `
-            };
-          })
-        }
-      },
-      args
-    });
-  }
+  render: () =>
+    html` <form>
+      <sd-switch required>Invalid</sd-switch>
+      <sd-button style="margin-top: 16px" type="submit">Submit</sd-button>
```

Comment:
Let's incorporate what @Vahid1919 did on `sd-radio-group`. Please do that together, thanks! :)